PayzaarTest/ProductStorage.cs

```diff
@@ -0,0 +1,112 @@
+﻿namespace PayzaarTest
+{
+    public class ProductStorage
```

Comment:
ProductUtils was renamed to ProductStorage. The reason for this is because the class went from static to public, it seemed to me that a class has the functionality to show available products according to time and that as a list of available products acted like a Storage. Hence, this behavior is best used when you are able to create instances.